src/docs/product/integrations/issue-tracking/jira/index.mdx

```diff
@@ -230,6 +230,10 @@ This issue typically occurs if you have an ad blocker blocking the conversation
 
 We don’t support custom required fields for the Jira integration. The only required fields we support are the ones that are pre-populated by us. If possible, you can edit your required fields in Jira, or as a workaround, you can create a ticket in Jira first and then link it in Sentry.
 
+### Issue Alert not Creating Jira Issues
+
+If your project configuration contains the "Priority" field, the dropdown will show every priority from all projects. Please ensure your alert's saved priority is available on your chosen project.
```

Comment:
It sounds like the problem is that the user selected a priority option that doesn't exist on this project. If so, can we say that directly?

```suggestion
This problem occurs when you've selected a priority option for the alert that doesn't exist for the given project. If your project configuration contains the "Priority" field, the dropdown will show every priority from **all** projects. Ensure your alert's saved priority is available on your chosen project.
```